### PR TITLE
Make code flake8-compatible, add "Styling" to docs.

### PR DESCRIPTION
**Pull request by bcipolli**
Hope this is useful! I wanted to pull some code (`setup.py` & `version.py`) into an existing flake8-compatible project, and I couldn't without modifying the code... so here we are :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,23 +101,43 @@ This indicates to you that a test has failed. In this case, the calculationg is
 As your code grows and becomes more complicated, you might develop new features that interact with your old features in all kinds of unexpected and surprising ways. As you develop new features of your code, keep running the tests, to make sure that you haven't broken the old features.  Keep writing new tests for your new code, and recording these tests in your testing scripts. That way, you can be confident that even as the software grows, it still keeps doing correctly at least the few things that are codified in the tests.
 
 
-### Documentation 
+### Styling
 
-Documenting your software is a good idea. Not only as a way to communicate to others about how to use the software, but also as a way of reminding yourself what the issues are that you faced, and how you dealt with them, in a few months/years, when you return to look at the code. 
+It is a good idea to follow the PEP8 standard for code formatting. Common code formatting
+makes code more readable, and using tools such as `flake` (which combines the tools
```
Comment (arokem) [inline README.md]: Is it `flake` or `flake8` here?

**Comment by arokem** [comment]
Do you want to add the flake8 thing into the travis configuration?

**Comment by bcipolli** [comment]
@arokem Fixed the README (thanks!), and added `flake8` to the Travis run.

File: shablona/__init__.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
-from .shablona import *
-from .version import __version__
\ No newline at end of file
+from .shablona import *  # noqa
```
Comment (arokem) [inline shablona/__init__.py:1]: Just noticed this: would you mind explaining (here, or elsewhere) what "noqa" means? I assume this is something that flake8 recognizes?

Comment (bcipolli) [inline shablona/__init__.py:1]: It means ignore the line for pep8/pyflakes (no quality assurance, I guess). pyflakes complains if you import things without explicitly using them. It's great at finding unused imports, variables, and variables used for assignment, but it complains with `from x import *` and things like this.

**Comment by arokem** [comment]
Thanks!

**Pull request merged**